client-mr/mr/src/main/java/org/apache/celeborn/mapreduce/v2/app/MRAppMasterWithCeleborn.java

```diff
@@ -197,5 +199,14 @@ public static void checkJobConf(JobConf conf) {
       // Make sure reduces are scheduled only after all map are completed.
       conf.setFloat(MRJobConfig.COMPLETED_MAPS_FOR_REDUCE_SLOWSTART, 1.0f);
     }
+    String masterEndpointsKey = HadoopUtils.MR_PREFIX + CelebornConf.MASTER_ENDPOINTS().key();
+    String masterEndpointsVal = conf.get(masterEndpointsKey);
+    if (masterEndpointsVal == null || masterEndpointsVal.isEmpty()) {
+      logger.info(
+          "MRAppMaster sets {} via environment variable {}.",
+          masterEndpointsKey,
+          MASTER_ENDPOINTS_ENV);
+      conf.set(masterEndpointsKey, ensureGetSysEnv(MASTER_ENDPOINTS_ENV));
```

Comment:
If the user does not set `mapreduce.celeborn.master.endpoints` and `CELEBORN_MASTER_ENDPOINTS`,
Is it possible to get output like this and get env is null exception?

```
MRAppMaster sets mapreduce.celeborn.master.endpoints via environment variable CELEBORN_MASTER_ENDPOINTS.
CELEBORN_MASTER_ENDPOINTS is null
```

Reply:
@cxzl25, `ensureGetSysEnv` could throw exception when `CELEBORN_MASTER_ENDPOINTS` is null.